README.md

```diff
@@ -1,111 +1,146 @@
-# IBM CIS Terraform Module
+# IBM Cloud Internet Services (CIS) Module
 
-Collection of modules that make it easier to provision Cloud Internet Services and its resources on IBM Cloud Platform:
+This module provisions an IBM Cloud Internet Services (CIS) instance and allows to add the following into the CIS instance.
+* Domain
```

Comment:
I would move up the domain config as it is a fundamental resource at the root module level when creating a cis instance. Alternative is to move the instance creation as a submodule and leave the root as empty (no module)

Reply:
1. `Root module as empty` - had discussion on this before. Generally in goldeneye modules we do not follow root modules as empty, so took this approach.
2. `domain can be moved into root module.` yes, it can be done. But when I created only instance, it worked. Infact if I do through UI, then also first step is instance creation and then domain and other things follow. So it is like - first step is instance creation and then configure the instance (via submodules) starting from domain. So I think keeping it as sub-module is fine and matches with the UI behavior.

Let me know your thoughts ...accordingly it can be changed.

Reply:
Yes, the main concern here is that the root module is just a simple wrapper around the provider resource, which is generally an anti-pattern. I think option 2 conceptually makes sense as I do not think the CIS instance is usable without domain config. Thinking further, perhaps the root could just call the domain submodule as opposed to moving up the logic. This would give a more compelling starting point at the root rather than just a thin wrapper.

Reply:
ok, understood the point. Made the changes as suggested.